reference/commands/custom_commands.rst

```diff
@@ -0,0 +1,139 @@
+.. _examples_extensions_custom_commands:
+
+Custom commands
+=================
+
+Since Conan 2.0, it's so easy to create your own Conan commands.
+
+Location and naming
+--------------------
+
+All the custom commands must be located in ``[YOUR_CONAN_HOME]/extensions/commands/`` folder. If _commands_ one is not created yet,
+you will have to do it. Those custom commands files must be Python files and start with the prefix ``cmd_[your_command_name].py``.
+The call to the custom commands is like any other existing Conan one: :command:`conan your_command_name`.
+
+
+Scoping
+++++++++++
+
+It's possible to have another folder layer to group some commands under the same topic.
+
+For instance:
+
+.. code-block:: text
+
+    [YOUR_CONAN_HOME]/extensions/commands/greet/
+        | - cmd_hello.py
+        | - cmd_bye.py
+
+The call to those commands change a little bit: :command:`conan [topic_name]:your_command_name`. Following the previous example:
```

Comment:
is it only one level, or could it be nested like `[topic1]:[topic2]:command`?

Reply:
The first sentence is clear I guess: `It's possible to have another folder layer to group some commands under the same topic.` Anyway, I can add a note to clarify that it's only one possible layer.